app/templates/src/unplanned-maintenance.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<!-- saved from url=(0078)https://surveys.onsdigital.uk/.extra/bres-service-temporarily-unavailable.html -->
+<html dir="ltr" lang="en-gb"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+
+
+
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+
+  <title>Service temporarily unavailable - ONS Business Surveys - Office for National Statistics (ONS)</title>
+
+  <meta name="description" content="ONS Business Surveys">
+
+  <!--[if (gt IE 9) | (IEMobile)]><!-->
+  <link rel="stylesheet" href="static/css/bres-responsive.css">
+  <!--<![endif]-->
+  <!--[if (lte IE 9) & (!IEMobile)]>
+  <link rel="stylesheet" href="/static/css/bres-fixed.css?q=4a6be5362d67d24cfb7ee2f106835e1084f8f847">
+  <![endif]-->
+  <!--[endif]-->
+
+  <link rel="stylesheet" href="/static/css/bres-site.css">
+  <link rel="stylesheet" href="/static/css/bres-lato.css">
+  <link rel="stylesheet" href="/static/css/bres-override.css">
+
+  <link href="https://surveys.onsdigital.uk/.extra/bres-favicon.ico" rel="Shortcut Icon">
+
+<style id="style-1-cropbar-clipper">/* Copyright 2014 Evernote Corporation. All rights reserved. */
```

Comment:
Can maybe get rid of this?  I don't think it's used, and if it is, it should live in a css file.

Reply:
Got rid of it